proposed/http-message.md

```diff
@@ -1053,8 +1053,8 @@ interface ServerRequestInterface extends RequestInterface
      * These values MAY be prepared from $_FILES or the message body during
      * instantiation, or MAY be injected via withUploadedFiles().
      *
-     * @return array An array tree of UploadedFileInterface instances; an empty
-     *     array MUST be returned if no data is present.
+     * @return UploadedFileInterface[] An array tree of UploadedFileInterface
```

Comment:
I actually specifically did *not* use this notation previously, as the structure is potentially a tree. While every leaf is an `UploadedFileInterface`, not every _value_ of the top-level array will necessarily be one.